components/esb-tools/plugins/org.wso2.integrationstudio.gmf.esb.diagram/src/org/wso2/integrationstudio/gmf/esb/diagram/custom/deserializer/NTLMMediatorDeserializer.java

```diff
@@ -0,0 +1,56 @@
+package org.wso2.integrationstudio.gmf.esb.diagram.custom.deserializer;
```

Comment:
Let's add the WSO2 license header for all the manually created files. Ex: .java, .xsd, etc.

Reply:
Added